.all-contributorsrc

```diff
@@ -1162,15 +1162,6 @@
       "contributions": [
         "doc"
       ]
-    },
```

Comment:
Make sure you don't accidentally delete this change, run `checkout main -- .all-contributorsrc` and then `git commit` to undo the change

Reply:
Sure, thanks.